### PR TITLE
Upgrade sbt

### PR DESCRIPTION
**Pull request by adamdecaf**

### CODE REVIEW

**Comment by adamdecaf** [comment]
retest this please

File: project/MultiScaladocBuild.scala [hunk 1]
```diff
@@ -29,34 +29,27 @@ object MultiScaladocBuild extends Build {
...
+          val akkaV = "2.3.8"
+          val sprayV = "1.3.2"
           Seq(
             "com.typesafe.akka" %% "akka-actor" % akkaV,
-            "io.spray" % "spray-routing" % sprayV,
-            "io.spray" % "spray-can" % sprayV,
-            "io.spray" %% "spray-json" % "1.2.5",
+            "io.spray" %% "spray-routing" % sprayV,
+            "io.spray" %% "spray-can" % sprayV,
+            "io.spray" %% "spray-json" % "1.3.0",
```
Comment (trentio) [inline project/MultiScaladocBuild.scala:41]: Does spray json not have a 1.3.2 release?

Comment (adamdecaf) [inline project/MultiScaladocBuild.scala:41]: spray-json versions are totally different from spray versions. it's confusing

Comment (trentio) [inline project/MultiScaladocBuild.scala:41]: Well thats super lame

Comment (adamdecaf) [inline project/MultiScaladocBuild.scala:41]: spray-json is just a separate project

**Comment by adamdecaf** [comment]
retest this please

**Pull request merged**